app/assets/javascripts/views/application/application_view.ts

```diff
@@ -81,6 +83,12 @@ class ApplicationViewCtrl extends PureViewCtrl<unknown, {
     await this.application.launch();
   }
 
+  async setAppVersion() {
+    const appVersion = packageJson.version;
```

Comment:
This value is already available globally: https://github.com/standardnotes/web/blob/develop/webpack.config.js#L17

Reply:
Didn't know that, thanks! Will update it.